packages/toolpad-app/src/runtime/ToolpadApp.tsx

```diff
@@ -437,6 +477,17 @@ function parseBindings(
           }
         }
       }
+
+      if (!isPageLayoutComponent(elm)) {
+        for (const [propName, argType] of Object.entries(layoutBoxArgTypes)) {
+          const binding =
+            elm.layout?.[propName as keyof typeof layoutBoxArgTypes] ||
+            appDom.createConst(argType?.defaultValue ?? undefined);
+          const bindingId = `${elm.id}.layout.${propName}`;
+          const scopePath = `${elm.name}.@layout.${propName}`;
```

Comment:
i want to make sure this scope path can't collide with any normal prop name. any other suggestions?

Reply:
It shouldn't be in teh scope, you can remove the scope path

Reply:
ok let's make these not be bindable for now - i'll just have to set the default value in a different way if i remember correctly